File: hw/Rodriguez/u3/hw29/hw29Python/Composite/controller/Clerk.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.Employee import Employee

class Clerk(Employee):
    def __init__(self, name):
        super().__init__(name, "Clerk")

    def add(self, employee):
        pass  

    def remove(self, employee):
        pass  

File: hw/ccedeno/u3/HW29/CompositePattern/src/main/java/ec/edu/espe/compositepattern/controller/Clerk.java

package ec.edu.espe.compositepattern.controller;

/**
 *
 * @author Andres Cedeno,Code Creators,DCCO-ESPE
 */
public class Clerk extends Employee{
    public Clerk(String aName){
        this();
        name = aName;
    }
    
    public void stateName(){
        super.stateName();
    }
    
    public Clerk(){
        title = "Clerk";
    }
}

